peoplehr/types.bal

```diff
@@ -928,3 +928,151 @@ public type ApplicantInformation record {|
     string Email?;
     *Authentication;
 |};
+
+# Represents the Custom Fields
+#
+# + ColumnName - The name of the custom field  
+# + ColumnValue - The value of the custom field
+public type Customfield record {
+    string ColumnName;
+    string ColumnValue;
+};
+
+# Represents Files Details
+#
+# + DocumentName - Name of the document name  
+# + Description - Description of the file  
+# + FileUrl - URL of the file  
+# + Category - File category  
+# + SignatureRequired - Whether a signature is required for the document
+public type File record {
+    string DocumentName; 
+    string Description; 
+    string FileUrl; 
+    string Category; 
+    boolean SignatureRequired;  
+};
+
+# Represents Files Details
+#
+# + DocumentName - Name of the document name  
+# + Description - Description of the file  
+# + File - The file content in Base64 string  
+# + DocumentCategory - File category  
+# + SignatureRequired - Whether a signature is required for the document
+public type NewFile record {
+    string DocumentName; 
+    string Description; 
+    string File; 
+    string DocumentCategory; 
+    boolean SignatureRequired;  
+};
+
+# Represents Employee Screen Details
+#
+# + ScreenId - ScreenId is the unique identifier of the employee custom screen  
+# + ScreenName - ScreenName is the name of the employee custom screen  
+# + TxnId - Field Description  
+# + Customfields - Custom fields details  
+# + Files - Files details
+public type EmployeeScreenDetail record {
+    int ScreenId?;
+    string ScreenName?;
+    int TxnId?;
+    Customfield[] Customfields?;
+    File[] Files?;
+};
+
+# Represents the response of `GetEmployeeScreenDetail` operation  
+#
+# + Result - The response of the operation
+public type EmployeeScreenDetailGetResponse record {|
+    *PeopleHRGenericResponse;
+    EmployeeScreenDetail[]|json Result;
```

Comment:
`Customer[] | record {}` is the better alternative. Had a chat with a Ballerina member.